pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/basicacmgkehubfeaturemembership/_generated_object_basicacmgkehubfeaturemembership.golden.yaml

```diff
@@ -25,6 +25,7 @@ spec:
         syncRev: HEAD
         syncWaitSecs: "20"
       sourceFormat: hierarchy
+    management: MANAGEMENT_AUTOMATIC
```

Comment:
Looks like `management` field has a default value. Does it mean that during the update, if the desired state doesn't have `spec.management`, but the live state has `management` and it is `MANAGEMENT_AUTOMATIC`, then there is no diff? If so, we should capture this logic in the Update function in the direct controller.